data_pipeline/vocab.py

```diff
@@ -3,7 +3,12 @@
 import os
 import pickle
 from copy import deepcopy
-
+import nltk
+from nltk.stem import WordNetLemmatizer
+from nltk.corpus import wordnet
+nltk.download('averaged_perceptron_tagger')
+nltk.download('wordnet')
+nltk.download('omw-1.4')
```

Comment:
What are these download statements? Can't they be imported?

Reply:
They need to be here, otherwise, it will not work.